### PR TITLE
[KernelGen][Nvidia] Add rad2deg operator with Triton kernel

### PR DESCRIPTION
**Pull request by XDYuanzhuLee**
## Summary
Adds a Triton kernel for `rad2deg`. Converts each element from angles in radians to degrees, including inplace variant `rad2deg_`.

## Testing
- Parametrized tests over POINTWISE_SHAPES and FLOAT_DTYPES
- Validated against reference on device via `to_reference(inp)`
- Tested on: Nvidia, Tianshu, Muxi, Ascend, Hygon

## Performance
Test command: `pytest benchmark/test_rad2deg.py --level core` (NVIDIA H20)

| Configuration | Torch Latency (ms) | Gems Latency (ms) | Speedup | TFLOPS |
|---|---|---|---|---|
| float16 [1073741824] | 1.100 | 2.538 | 0.433 | 0.423 |
| float16 [64,64] | 0.006 | 0.017 | 0.329 | 0.000 |
| float16 [4096,4096] | 0.023 | 0.047 | 0.496 | 0.358 |
| float16 [64,512,512] | 0.023 | 0.047 | 0.500 | 0.360 |
| float16 [1024,1024,1024] | 1.107 | 2.185 | 0.507 | 0.491 |
| float32 [1073741824] | 2.030 | 2.730 | 0.744 | 0.393 |
| float32 [64,64] | 0.005 | 0.017 | 0.328 | 0.000 |
| float32 [4096,4096] | 0.038 | 0.054 | 0.698 | 0.311 |
| float32 [64,512,512] | 0.038 | 0.055 | 0.689 | 0.308 |
| float32 [1024,1024,1024] | 2.072 | 2.734 | 0.758 | 0.393 |
| bfloat16 [1073741824] | 1.116 | 2.188 | 0.510 | 0.491 |
| bfloat16 [64,64] | 0.005 | 0.018 | 0.303 | 0.000 |
| bfloat16 [4096,4096] | 0.023 | 0.047 | 0.496 | 0.358 |
| bfloat16 [64,512,512] | 0.024 | 0.047 | 0.504 | 0.358 |
| bfloat16 [1024,1024,1024] | 1.118 | 2.219 | 0.504 | 0.484 |
| **Arithmetic Mean** | — | — | **0.520** | — |

## Multi-backend Testing
| Backend | Accuracy Test | Benchmark | Speedup (mean) | Notes |
|---|---|---|---|---|
| Nvidia (H20) | PASS | PASS (15 cases, --level core) | 0.520 | Primary |
| Tianshu | PASS | PASS (42 cases) | 1.011 | — |
| Muxi | PASS | PASS (42 cases) | 1.013 | — |
| Ascend | PASS | N/A | — | — |
| Hygon | PASS | PASS (42 cases) | 1.052 | — |

## Files Changed
- `src/flag_gems/ops/rad2deg.py`: Triton kernel implementation
- `tests/test_rad2deg.py`: Accuracy test
- `benchmark/test_rad2deg.py`: Performance benchmark
- `src/flag_gems/ops/__init__.py`: Register import and `__all__`
- `src/flag_gems/__init__.py`: Register to `_FULL_CONFIG`
- `conf/operators.yaml`: Add operator entry (kind: Math, stage: alpha 5.1)

### CODE REVIEW

File: conf/operators.yaml [hunk 1]
```diff
@@ -4496,6 +4496,20 @@ ops:
       - Tensor
     stages:
       - stable: '2.2'
+  - id: rad2deg
+    description: |
+      Converts each element from angles in radians to degrees.
+    for:
+      - rad2deg
```
Comment (Dongxu-H) [inline conf/operators.yaml:4562]: split into 2 kernels

File: benchmark/test_rad2deg.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import pytest
+import torch
+
+from . import base, consts
+
+
+@pytest.mark.rad2deg
```
Comment (Dongxu-H) [inline benchmark/test_rad2deg.py:7]: rad2deg_ is missing

File: conf/operators.yaml (hunk 1 above)
Comment (XDYuanzhuLee) [reply conf/operators.yaml:4562]: Done. Split into `rad2deg` + `rad2deg_` entries in yaml.

File: benchmark/test_rad2deg.py (hunk 2 above)
Comment (XDYuanzhuLee) [reply benchmark/test_rad2deg.py:7]: Done. Added `test_rad2deg_()` benchmark with inplace op.

File: conf/operators.yaml (hunk 1 above)
Comment (XDYuanzhuLee) [reply conf/operators.yaml:4562]: Done. Split into `rad2deg` + `rad2deg_` entries in yaml.

File: benchmark/test_rad2deg.py (hunk 2 above)
Comment (XDYuanzhuLee) [reply benchmark/test_rad2deg.py:7]: Done. Added `test_rad2deg_()` benchmark with inplace op.

**Review by w1120029931-bit — APPROVED** [review]
